include/flatbuffers/flatbuffers.h

```diff
@@ -2748,7 +2748,8 @@ inline const char * const *ElementaryTypeNames() {
 struct TypeCode {
   uint16_t base_type : 4;  // ElementaryType
   uint16_t is_vector : 1;
-  int16_t sequence_ref : 11;  // Index into type_refs below, or -1 for none.
+  uint16_t is_array : 1;
```

Comment:
we could actually just rename `is_vector` into `is_repeating` or so, since vector is only used inside tables and array only inside structs? Add a comment saying this means either vector or array.